src/classes/HH_Container_LCTRL.cls

```diff
@@ -37,6 +37,28 @@
 public class HH_Container_LCTRL {
     
     /*******************************************************************************************************
+    * @description trick the packaging spider to pickup references to custom labels only used by our
+    * lightning components.
+    */
+    public static void packageLabels() {
```

Comment:
I'm going to pull this branch down and review the diff on my local machine.  Will merge after I re-review.